default/stringtables/en.txt

```diff
@@ -4219,6 +4219,39 @@ Next are icons showing the player empire total [[encyclopedia INDUSTRY_TITLE]],
 
 On the right hand side of the control bar are a number of icons for toggling display of other significant UI windows. Tooltips are available for these.'''
 
+DESIGN_WINDOW_ARTICLE_TITLE
+Design Window
+DESIGN_WINDOW_ARTICLE_TEXT
+'''The design window is for viewing or altering the designs of monsters or ships.
+
+There are four sub windows:  Design Starts, Ship Parts, Design Details and the Pedia.
+
+
+Design Starts has four tabs: empty Hulls, Finished designs, Saved designs from previous games and Monster designs for inspection.  Drag or double click hulls, designs or monsters to edit/examine them on the workbench.
+
+Finished designs appear in the same order as on the Production Window.  Drag and drop designs within the list to change the order.  Delete designs with Obsolete from the context menu.  Ships already queued for production will continue to be built with the old design.  Save designs to disk for future games with Save from the context menu.
+
+Right click Saved designs and copy a single or all saved designs to the empire's known designs.
+
+
+Ship Parts displays parts. Parts whose associated technology has been researched may be used in current designs.  Other parts can not be used because either the associated technology has not been researched or the part is only available to monsters.
+
+The Available and Unavailable filters on Design Starts and Ship Parts toggle the display of parts that the empire can currently use or that are restricted.  Researching the associated technology makes parts available.  Some parts will never be available because they are only used for monsters.
```

Comment:
I'd just write something like "Some unavailable parts may be unlocked by researching associated technologies."